string forward_back # 直進方向 ("forward" or "back")
float64 distance # 直進する距離 (単位: m)
string left_right # 回転方向（"left" or "right"）
float64 angle   # 回転する角度（単位: rad）
---
bool res    # サービスの成功/失敗を表すフラグ